content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -69,10 +69,10 @@ For more details please see the [Network Plugin Requirements](/docs/concepts/ext
 ## Check required ports
 These
 [required ports](/docs/reference/ports-and-protocols/)
-need to be open in order for Kubernetes components to communicate with each other. You can use telnet to check if a port is open. For example:
+need to be open in order for Kubernetes components to communicate with each other. You can use NetCat to check if a port is open. For example:
```

Comment:
```suggestion
need to be open in order for Kubernetes components to communicate with each other. You can use tools like netcat to check if a port is open. For example:
```

netcat is stylized lowercase.
https://en.wikipedia.org/wiki/Netcat

> $ nc -h
> OpenBSD netcat (Debian patchlevel 1.187-1ubuntu0.1)

is nc available in most popular distros without pre-installing with a package manager?